lib/flop.ex

```diff
@@ -445,8 +449,7 @@ defmodule Flop do
   @doc since: "0.6.0"
   @spec all(Queryable.t(), Flop.t(), [option()]) :: [any]
   def all(q, flop, opts \\ []) do
-    repo = opts[:repo] || default_repo() || raise no_repo_error("all")
-    apply(repo, :all, [query(q, flop)])
+    apply_on_repo(:all, [query(q, flop)], opts)
```

Comment:
I think this should be `apply_on_repo(:all, "all", [query(q, flop)], opts)`, if you don't want to change the raised error.

Reply:
I think `no_repo_error` converts the `:all` implicitly to a string, so it was ok, but I can make the option explicit as well 👍